src/components/modals/WalletListMenuModal.tsx

```diff
@@ -210,24 +203,19 @@ export function WalletListMenuModal(props: Props) {
         </View>
       )}
 
-      <View style={styles.scrollViewContainer}>
-        <ScrollView contentContainerStyle={styles.scrollViewPadding}>
-          {options.map((option: Option) => (
-            <TouchableOpacity key={option.value} onPress={() => optionAction(option.value)} style={styles.row}>
-              <AntDesignIcon
-                // @ts-expect-error
-                name={icons[option.value] ?? 'arrowsalt'} // for split keys like splitBCH, splitETH, etc.
-                size={theme.rem(1)}
-                style={option.value === 'delete' ? [styles.optionIcon, styles.warningColor] : styles.optionIcon}
-              />
-              <Text style={option.value === 'delete' ? [styles.optionText, styles.warningColor] : styles.optionText}>{option.label}</Text>
-            </TouchableOpacity>
-          ))}
-        </ScrollView>
-      </View>
-      <LinearGradient style={styles.modalCloseButton} colors={[xButtonTopColor, xButtonBottomColor]} start={xButtonGradientStart} end={xButtonGradientEnd}>
-        <ModalCloseArrow onPress={handleCancel} />
-      </LinearGradient>
+      <ModalScrollArea onCancel={handleCancel}>
+        {options.map((option: Option) => (
+          <TouchableOpacity key={option.value} onPress={() => optionAction(option.value)} style={styles.row}>
+            <AntDesignIcon
+              // @ts-expect-error
```

Comment:
Can this be fixed?

Reply:
Look at the PR with whitespace hidden. It's been this way for years, this PR is not about fixing it.